packages/browser-wallet-api-helpers/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## Unreleased
+
+### Breaking changes
+
+-   Removed `getGrpcClient`. As an alternative, `grpcTransport` can be used with `ConcordiumGRPCClient` from `@concordium/web-sdk`.
+-   Make dependency to `@concordium/web-sdk` a peer dependency (^7.0).
+    This will require consuming apps to add an explicit dependency to the library with its own version constraints.
+
+### Added
+
+-   `grpcTransport` getter for getting a gprc transport layer for constructing a GRPC client which uses the node settings from the wallet.
```

Comment:
From my understanding, it is still valuable to users to be able to use the node from the wallet to be able to easily query the network users are connected to. What we wanted to avoid was: not being able to introduce changes to the grpc client in the SDK, as this would in turn break any dapp that did not update its code to be compatible with whichever client was used by the API injected by the wallet (which could in theory be any version of the API).